zstacklib/zstacklib/utils/lvm.py

```
@@ -1463,30 +1463,22 @@ def extend_lv(path, extend_size, skip_if_sufficient=False):
 
-    current_size = int(get_lv_size(path))
-
     if extend_thin_by_specified_size:
         v_size = linux.qcow2_virtualsize(path)
-        if size + cmd.addons[thinProvisioningInitializeSize] > v_size:
+        thin_init_size = cmd.addons[thinProvisioningInitializeSize]
+        if size + thin_init_size > v_size:
             size = v_size
         else:
-            size = size + cmd.addons[thinProvisioningInitializeSize]
+            size = size + thin_init_size
```

Ruff (0.12.2)

Use ternary operator `size = v_size if size + thin_init_size > v_size else size + thin_init_size` instead of `if`-`else`-block

Replace `if`-`else`-block with `size = v_size if size + thin_init_size > v_size else size + thin_init_size`

(SIM108)